Jenkinsfile-build

```diff
@@ -1,5 +1,13 @@
 def versionTag = "v${env.BUILD_NUMBER}"
 
+def setBuildStatus(String status, String desc) {
```

Comment:
Loki is [adding this function to the shared library](https://github.com/nationalarchives/tdr-jenkinslib/pull/1), so it might be worth waiting for that to be available and then updating this PR.

Reply:
Sorry this is marked as draft. Using this PR to test the global library functionality with Loki